tests/integration/test_cubepart.py

```diff
@@ -1016,9 +1058,28 @@ def test_it_can_sort_by_marginal_with_nan_in_body(self):
             [2.45356177, 2.11838791, 2.0, 1.97, 1.74213625, np.nan], nan_ok=True
         )
 
-    def test_it_can_fix_order_of_subvars_identified_by_bogus_id(self):
+    @pytest.mark.parametrize(
+        "dimension, expceted_labels",
+        (
+            ("rows_dimension", ["Finland", "Sweden", "Norway", "Denmark", "Iceland"]),
+            (
+                "columns_dimension",
+                [
+                    "Chitarra",
+                    "Quadrefiore",
+                    "Orecchiette",
+                    "Fileja",
+                    "Bucatini",
+                    "Boccoli",
+                ],
+            ),
+        ),
+    )
+    def test_it_can_fix_order_of_subvars_identified_by_bogus_id(
```

Comment:
I kind of think this test should be split out rather than parametrized - the test name is about MR bogus ids (the integers rather than MR alias), but since it is MRxCAT, your new column based test is actually using the real category ids.

Also, by "luck" there's no difference between "fixed category 3 to the bottom" because apparently category 3 is "Boccoli" which would have been last anyways.